### ISSUE TITLE
v2 Cannot find module './PgCatalog'

### ISSUE THREAD

**Issue by tim-field** [calebmer/postgraphql#156]
Hello

Trying to use v2 but I'm getting this error when using it as a library 

```

import postgraphql from 'postgraphql'
import express from 'express'
...
let app = express()
app.use('/graphql', postgraphql(DB_STRING, [DB_SCHEMA]))

```

When I try and run the server I get.

```
Error: Cannot find module './PgCatalog'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/srv/www/qstoolbox-api/node_modules/postgraphql/build/postgres/introspection/introspectDatabase.js:13:21)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
```

**Comment by calebmer** [comment]
What version? I’ve posted a couple today :blush:

The latest is `2.0.0-beta.6`.

**Comment by tim-field** [comment]
Sorry should have said, yeah this is with 

2.0.0-beta.6

**Comment by tim-field** [comment]
also breaks with 2.0.0-beta.5 but its ok with 2.0.0-beta.4

**Comment by calebmer** [comment]
My bad, fixed in `postgraphql@2.0.0-beta.7` :+1:
